src/target/mips32_pracc.h

```diff
@@ -114,7 +114,7 @@ int mips32_cp0_read(struct mips_ejtag *ejtag_info,
 int mips32_cp0_write(struct mips_ejtag *ejtag_info,
 		uint32_t val, uint32_t cp0_reg, uint32_t cp0_sel);
 
-inline void pracc_swap16_array(struct mips_ejtag *ejtag_info, uint32_t *buf, int count)
+static inline void pracc_swap16_array(struct mips_ejtag *ejtag_info, uint32_t *buf, int count)
```

Comment:
Mainline OpenOCD does not Just Work with this. On some compilers this fix is required, but it hasn't made it to mainline yet. I got this fix from OpenOCD's gerrit, where it has been approved but not yet merged.